packages/orbit-components/src/Card/CardSection/index.tsx

```diff
@@ -58,26 +80,20 @@ export default function CardSection({
 
   return (
     // Needs to capture bubbled click events from the <button> below
-    // eslint-disable-next-line jsx-a11y/no-static-element-interactions
     <div
       className={cx(
-        "duration-fast lm:border-x border-b transition-all ease-in-out",
+        "duration-fast lm:border-x relative border-b transition-all ease-in-out",
         opened && "my-200 rounded-100 shadow-level2 [&+*]:border-t",
-        onClick != null && "hover:bg-white-normal-hover cursor-pointer",
+        onClick && !expandable && "hover:bg-white-normal-hover",
+        onClick &&
+          "has-[.orbit-card-wrapper-button:focus]:focus-within:rounded-100 has-[.orbit-card-wrapper-button:focus]:focus-within:outline-blue-normal has-[.orbit-card-wrapper-button:focus]:focus-within:outline has-[.orbit-card-wrapper-button:focus]:focus-within:outline-2",
       )}
       data-test={dataTest}
-      role={onClick == null ? undefined : "button"}
-      // See comment above
-      // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
-      tabIndex={onClick == null ? undefined : 0}
-      onClick={onClick}
```

Comment:
As you removed this, now when card is expandable, `onClick` is not fired when clicked on the header, is that desired change?

Reply:
When the Card is expandable, toggling the card should be prioritized over `onClick`, onClick is working for "children" content clicking. I think it was working like that before. 🤔

Reply:
Master:


https://github.com/user-attachments/assets/b5758d77-4110-448d-8485-5984e591236e

Your branch:


https://github.com/user-attachments/assets/a4d89f84-1d01-4517-ba5f-d5b8d86f6fd1